Dockerfile

```diff
@@ -41,15 +41,15 @@ RUN rpm -e --nodeps tzdata             && \
     microdnf install --nodocs -y $deps && \
     gem update bundler                 && \
     microdnf clean all -y              && \
-    chown 1001:root ./
+    chown 1001:root ./                 && \
+    install -d -m 0775 -o 1001 ./tmp ./log
```

Comment:
There is [tmp/storage](https://github.com/RedHatInsights/compliance-backend/tree/master/tmp/storage) in the repo. Wouldn't that affect this `install` call?

Reply:
Check the `Dockerignore` :wink: but that can be deleted anyway, will do a followup PR.